projects/data_collection/collect_h5.py

```diff
@@ -79,7 +79,7 @@ def record_keyframe(self):
 @click.option("--dir-path", default="./H5s/", help="Path of root data directory")
 def main(task_name, dir_path):
     em = EpisodeManager(task_name, dir_path)
-    rate = rospy.Rate(10)
+    rate = rospy.Rate(20)
```

Comment:
why?

Reply:
Trial-and-error for smoother robot execution until I found the fix with Sam.